src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -841,6 +841,9 @@ def load_arguments(self, _):
         c.argument('ppg_type', options_list=['--type', '-t'], help="The type of the proximity placement group. Allowed values: Standard.")
         c.argument('tags', tags_type)
 
+    with self.argument_context('ppg show', min_api='2019-07-01') as c:
+        c.argument('include_colocation_status', options_list=['--colocation-status'], arg_type=get_three_state_flag(), help='Enable fetching the colocation status of all the resources in the proximity placement group. Default is false.')
```

Comment:
Remove options_list to avoid breaking change? In addition, the name `include_colocation_status` seems more clear. I suggest you to use `action='store_true'` since include_colocation_status is not a property of the resource, it's more like a command switch.

Reply:
I agree. As `include_colocation_status` doesn't go to a property of the resource.

Reply:
Good idea. Keep the original name.
Right. It's not a property.
Fixed.